include/systems/parameter_multiaccessor.h

```diff
@@ -75,7 +75,8 @@ class ParameterMultiAccessor : public ParameterAccessor<T>
   {
     libmesh_assert(!_accessors.empty());
 #ifndef NDEBUG
-    T& val = _accessors[0]->get();
+    // Compare other values to the last one we'll change
+    T& val = _accessors[_accessors.size()-1]->get();
```

Comment:
If accessors.size() == 0, the assertion above will fail, and .back() won't be defined anyway.  It certainly would have been more self-documenting, though...